src/pages/public_registration/preview_file/PreviewOfficeFile.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled from 'styled-components';
+import { CommonPreviewProps } from './PreviewFile';
+
+const StyledIframe = styled.iframe`
+  width: 100%;
+  height: 25rem;
+`;
+
+interface PreviewOfficeFileProps extends CommonPreviewProps {
+  iframeTitle: string;
+}
+
+export const PreviewOfficeFile = ({ url, iframeTitle, ...props }: PreviewOfficeFileProps) => {
+  const officeViewerUrl = `https://view.officeapps.live.com/op/embed.aspx?src=${encodeURIComponent(url)}`;
```

Comment:
informerer JEG/Hege om denne, og om dette er noe som burde inn i personvernerklæringen for det går noen kall dit nå (de brukte samme løsning i DLR)